UI/MainMenu/LoadGameScreen.gd

```diff
@@ -19,39 +22,34 @@ func _ready():
 	if OS.get_name() == "HTML5" and OS.has_feature("JavaScript"):
 		_define_js()
 
-func updateSaves():
-	Util.delete_children(savesContainer)
-	
+func updateSaves(): #Argh!!! It's was so unefficient...
```

Comment:
It was clean code though. There is no need to optimize menu stuff, that will just make it harder to extend

Reply:
Your approach uses constant deleting and creating new data in memory, it's slows processing.